src/compositor.rs

```diff
@@ -276,14 +279,20 @@ impl Builder {
         self
     }
 
-
     /// Decide whether or not to enable the screenshooter protocol
     /// extension.
     pub fn screenshooter(mut self, screenshooter: bool) -> Self {
         self.screenshooter = screenshooter;
         self
     }
 
+    /// Decide whether or not to enable the virtual keyboard protocol
+    /// extension.
+    pub fn virtual_keyboard_manager(mut self, virtual_keyboard_manager: bool) -> Self {
```

Comment:
The manager has an event that should be listened to: `new_virtual_keyboard`. When it's triggered we should be able to hook in code with a callback.

As an example of using this callback, [here is it being used in wlroots' rootston code](https://github.com/swaywm/wlroots/blob/master/rootston/virtual_keyboard.c). Note that because it accesses the `wlr_seat` pointer from the `wlr_virtual_keyboard_v1` (which is what will need to be passed into that callback) this will also need to be wrapped.